src/lib/rule-finder.js

```diff
@@ -108,6 +108,6 @@ function RuleFinder(specifiedFile, options) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, options = {}) {
+module.exports = function (specifiedFile, options) {
```

Comment:
@ta2edchimp Can you just revert my last commit, or do you need me to submit a new PR with this change?